src/components/signon/SignOn.tsx

```diff
@@ -367,10 +366,10 @@ const SignOn = ({ navigation }: SignOnProps) => {
       setPassword('')
 
       if (isSignin) {
-        remindUserToTurnOnNotifications(setPushNotificationsReminderVisible)
+        remindUserToTurnOnNotifications(dispatch)
```

Comment:
is this ok?

Reply:
Yup this change is tested, just a minor refactor to keep things dry